test/evergreen.yml

```diff
@@ -468,7 +469,7 @@ functions:
           pip3 install google-cloud-storage
 
           # Run Python testing for all tiered tests.
-          python3 ../test/suite/run.py -j $(nproc) ${tiered_storage_test_name}
+          python3 ../test/suite/run.py ${ignore_stdout|} -j $(nproc) ${tiered_storage_test_name}
```

Comment:
The default value of a variable is already empty. So we can just do ${ignore_stdout}

Reply:
Done in [dfd1c1b](https://github.com/wiredtiger/wiredtiger/pull/10839/commits/dfd1c1b6df6ac36c0444629f189bf7aed917563e)